java/src/gov/usgs/consumerclient/ConsumerClient.java

```diff
@@ -389,6 +412,15 @@ public static void main(String[] args) {
 				// log exception
 				logger.error(e.toString());
 			}	
+
+			// sleep a little while before the next loop
+			try {
```

Comment:
Why is it sleeping 100ms after each processed message?

Reply:
100's always been the number I used for sleeps within a loop.

Reply:
I don't understand how you could sleep 100 each time through the loop and then grab a new timestamp and get the same time value if it includes milliseconds.  I could see it, if your sleep time was less than a tick or whatever the OS allocates for a timeslice, but 100ms should be way longer than that.  Are you not grabbing a new time value, or are you using a function that only calculates the seconds?